src/stats/common.ts

```diff
@@ -83,6 +83,9 @@ export interface OverallType extends PlayerIndexedType {
   successfulConversions: RatioType;
   inputsPerMinute: RatioType;
   digitalInputsPerMinute: RatioType;
+  joystickDistanceTraveled: number;
```

Comment:
Instead of having these all at the top level, make a joystick section and create a new JoystickType, similar to the InputCountsType in usage. Doesn't have to be called JoystickType, I just don't have a better suggestion atm.